src/bundle/Resources/config/routing.yml

```
@@ -583,6 +583,15 @@ ezpublish_rest_loadContentTypeFieldDefinition:
     requirements:
         contentTypeId: \d+
         fieldDefinitionId: \d+
+        
+ezpublish_rest_loadContentTypeFieldDefinitionByIdentifier:
+    path: /content/types/{contentTypeId}/fieldDefinition/{fieldDefinitionIdentifier}
+    defaults:
+        _controller: ezpublish_rest.controller.content_type:loadContentTypeFieldDefinitionByIdentifier
```

Use newer Symfony notation.

For controller declaration [it has become deprecated](https://symfony.com/blog/new-in-symfony-4-1-deprecated-the-bundle-notation) to reference them using a single colon.

```suggestion
    controller: ezpublish_rest.controller.content_type::loadContentTypeFieldDefinitionByIdentifier
```